src/dataconnect/schemaMigration.ts

```diff
@@ -43,6 +44,15 @@ export async function diffSchema(
   );
   let diffs: Diff[] = [];
 
+  await setupIAMUsers(instanceId, databaseId, options);
```

Comment:
Nit: This chunk is (nearly) exactly repeated between the two functions, consider DRYing out with a helper function